Sources/Interaction/Style/InteractorStyleImage/index.js

```diff
@@ -52,7 +52,11 @@ function vtkInteractorStyleImage(publicAPI, model) {
       publicAPI.startWindowLevel();
     } else if (model.interactionMode === 'IMAGE3D' && callData.shiftKey) {
       // If shift is held down, do a rotation
```

Comment:
This comment should be updated to reflect this change in controls, or just remove the comment entirely since the code is self-explanatory.

Do we have any other docs that reference this? If so, we would need to update them.